optimize/util/optimize-commons/src/main/java/io/camunda/optimize/rest/constants/RestConstants.java

```diff
@@ -14,7 +14,7 @@ public final class RestConstants {
   public static final String X_OPTIMIZE_CLIENT_TIMEZONE = "X-Optimize-Client-Timezone";
   public static final String X_OPTIMIZE_CLIENT_LOCALE = "X-Optimize-Client-Locale";
   public static final String AUTH_COOKIE_TOKEN_VALUE_PREFIX = "Bearer ";
-  public static final String OPTIMIZE_AUTHORIZATION = "X-Optimize-Authorization";
+  public static final String OPTIMIZE_AUTHORIZATION_PREFIX = "X-Optimize-Authorization_";
```

Comment:
Just one small detail. This is generating the name with 2 underscores, like `X-Optimize-Authorization__1`. If this is what you wanted, then it's fine already. I'm pointing it out because I saw that you do
```
OPTIMIZE_AUTHORIZATION_PREFIX + "_"
```
in one place, and the result is this:
<img width="233" alt="image" src="https://github.com/user-attachments/assets/c45e94b8-1a2f-4c09-8c8e-d3b6d9e0e938" />

I am pre-apprving it anyways because it's working on my computer

Reply:
I have fixed the double underscoring, nice catch 👍